spec/models/bathroom_spec.rb

```diff
@@ -22,4 +23,20 @@
       end
     end
   end
+  describe '.avg_rating' do
+    context 'there are reviews with ratings' do
+      it 'should calulate average rating for bathroom' do
+        bathroom = FactoryGirl.create(:bathroom)
+        review =  FactoryGirl.create(:review, bathroom: bathroom)
```

Comment:
Useless assignment to variable - `review`. Did you mean `review2`?<br>Unnecessary spacing detected.